rdflib/namespace.py

```diff
@@ -95,11 +95,6 @@ def __new__(cls, value):
             rt = unicode.__new__(cls, value, 'utf-8')
         return rt
 
-
-    @property
-    def title(self):
-        return URIRef(self + 'title')
-
```

Comment:
So it should be added back (along with a test) since existing code will break otherwise.